### ISSUE TITLE
How to or feature request: Piping multiple tasks into each other

### ISSUE THREAD

**Issue by crackingTest** [s3inlc/hashtopussy#336]
Your current Server version located at the bottom of any hashtopussy webpage.
0.4.3 according to banner, latest dev branch as of Feb 27 2018

Current Client version
0.4.3 C#
Your current Hashcat version
4.1.0
The exact task command you are trying to run.
I need to be able to combine multiple hashcat commands into one. On a local system, this is simple - you use hashcat's --stdout flag, a pipe and then use that as input for the next hashcat command. However, the pipe character('|') is apparently not allowed in hashtopussy, which makes sense since it's a web client. But could this be added as perhaps a kind of super task? If this is already possible, sorry, I'm rather new to hashtopussy, and I'd appreciate some guidance. 

Debug output from the client by running "hashtopussy.exe debug"
It never gets to that state. I mean, I can dispatch a single command to my HTP clients just fine, but that's not what I need.

Describe your problem in as much detail as possible " It's broke " is not a description.
The issue's above. The reason I need to do this is I need to apply some rules to a wordlist in a very specific order, which I don't think can be done in one command. 

Thanks in advance folks!

**Comment by winxp5421** [comment]
We need to know how large the entire task is in order to create chunks of the task and send to multiple agents. What you are asking for is quite impossible on the hashtopussy platform.

Here is a low level look at why this is not exactly feasible. 

Gary has 20% of the company funds. The amount of company funds is unknown. How much company money does Gary have in his possession? 

The only answer you can say is Gary has 20% we cant actually equate this into an actual number without first knowing how much money the company has but, seen as we don't know how much money the company has its not possible to find out.

Long story short. We HAVE to have a finite value that states the task's keyspace is X large. without it its not possible to split the task into chunks and distribute it between machines.
